spec/lib/mcb/commands/users/grant_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'mcb_helper'
 
-describe 'mcb users grant_access_to_provider' do
-  def grant_access_to_provider(provider_code, commands)
+describe 'mcb users grant' do
```

Comment:
```suggestion
describe 'mcb users grant access' do
```

`mcb users grant access when the user exists and already has access to the provider`
vs
`mcb users grant when the user exists and already has access to the provider`

Reply:
`grant_access` maybe.

Reply:
`mcb users grant` is exactly what you type in the terminal, so I'm inclined to leave it?